shared/teams/team/member/container.js

```diff
@@ -98,7 +96,7 @@ const mergeProps = (stateProps: StateProps, dispatchProps: DispatchProps) => {
   // otherwise you just need to be an admin
   let admin = user.type === 'owner' ? you.type === 'owner' : you.type === 'owner' || you.type === 'admin'
   if (stateProps.teamname.includes('.')) {
-    admin = admin || stateProps._implicitAdminUsernames.contains(you.username || '')
+    admin = admin || stateProps.yourOperations.manageMembers
```

Comment:
It seems like this entire block should be replaced, right? Why are we still doing things like testing owner and admin directly?